Formula/openimageio.rb

```diff
@@ -0,0 +1,72 @@
+class Openimageio < Formula
+  desc "Library for reading, processing and writing images"
+  homepage "http://openimageio.org/"
+  url "https://github.com/OpenImageIO/oiio/archive/Release-1.8.7.tar.gz"
+  sha256 "ed23ecdd3e4037a411c69a6a681273d01d4a2fbcc223eac86daba77dd03fc5ed"
+  head "https://github.com/OpenImageIO/oiio.git"
+
+  bottle do
+    cellar :any
+    sha256 "77bdd8b9b3e1b1ca6abd7eaa98227b0794b98b43fbacb24348774c4de10a4570" => :high_sierra
+    sha256 "b50f595ad597de8774ac929917d0371d2dca702349e3e4538250dc9fcf0050fd" => :sierra
+    sha256 "02adc761d2d21864e480795a4ef527de7852401770dec5dac31ea99872347001" => :el_capitan
+  end
+
+  option "without-python", "Build without python2 support"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "boost"
+  depends_on "ffmpeg"
+  depends_on "freetype"
+  depends_on "giflib"
+  depends_on "ilmbase"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "libraw"
+  depends_on "libtiff"
+  depends_on "opencolorio"
+  depends_on "openexr"
+  depends_on "openjpeg"
+  depends_on "openssl"
+  depends_on "webp"
+  depends_on "python3" => :optional
+
+  depends_on "boost-python" if build.with?("python") && build.without?("python3")
+  depends_on "boost-python3" if build.with? "python3"
+
+  def install
+    args = std_cmake_args
+    args << "-DSELF_CONTAINED_INSTALL_TREE=ON" # stub from eluded make wrapper
+
+    args << "-DEMBEDPLUGINS=ON" << "-DUSE_OPENGL=OFF" << "-DUSE_QT=OFF" \
+         << "-DUSE_JPEGTURBO=OFF" << "-DUSE_NUKE=OFF" << "-DUSE_FIELD3D=OFF" \
+         << "-DUSE_OPENCV=OFF" << "-DUSE_PTEX=OFF"
+
+    args << "-DUSE_PYTHON=OFF" if build.without?("python") || build.with?("python3")
+
+    # CMake picks up the system's python dylib, even if we have a brewed one.
+    if build.with? "python3"
+      py3ver = Language::Python.major_minor_version "python3"
+      py3prefix = `python3-config --prefix`.chomp
```

Comment:
probably follow The Law of @DomT4 regarding backticks :)

Reply:
Not needed since we can use `Formula["python3"].opt_frameworks/"Python.framework/Versions/#{py3ver}"` here.

Reply:
:)

Reply:
Done